app/jobs/calendar_importer/parsers/out_savvy.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# In order for a parser to be recognized, it must be added
+# to the PARSERS constant list in app/models/calendar_parser.rb.
+# Parent parser classes should not be added.
+
+module CalendarImporter::Parsers
+  class OutSavvy < Base
+    NAME = 'OutSavvy'
+    KEY = 'out-savvy'
+    DOMAINS = %w[www.outsavvy.com].freeze
```

Comment:
do we need this www?

Reply:
This is just the thing that is shown to the user on the calendar page